secure-boot-example/README.md

```diff
@@ -1,8 +1,153 @@
-This directory contains an example secure boot image signed with the example private key in this directory.
+# Secure-boot quickstart
+
+## Overview
+Secure-boot is a mechanism for verifying the integrity of the kernel image and 
+other files required during boot by storing them in a signed disk image.
+These files include the GPU firmware (start.elf etc), Kernel, initrd, device-tree tree
+and overlays.
+Secure-boot does require any particular OS and does not extend into the OS
```

Comment:
does require -> does NOT? require

Reply:
"does not extend into the OS" I might suggest rewording this - does not require any support from the OS?